policies/lib/all_rekor_attestations.rego

```diff
@@ -0,0 +1,26 @@
+package lib
+
+#
+# A list of all attestations from rekor data
+#
+# The result is a list of hashes with three keys,
+# "rekor_host", "log_index", and "data".
+#
+all_rekor_attestations = attestations {
+	attestations := [attestation |
+		rekor_hosts := data.rekor[rekor_host]
+		log_entry := rekor_hosts.index[log_index].entry
+		attestation := {
+			"rekor_host": rekor_host,
+			"log_index": log_index,
+			"data": decode_attestation(log_entry.Attestation),
```

Comment:
Oh nice!

Reply:
It is, but I suppose we won't need it soon.. :)